linked_list/test_linked_list.py

```diff
@@ -0,0 +1,54 @@
+from linked_list import LinkedList as ll
+import pytest as pytest
```

Comment:
You only need to import these things if you use them in your tests. Since you are defining any fixtures or testing for exceptions, you do not need to import `pytest`. Since you are not using your `LinkedList` constructor in this file, you do not need to import `linked_list`.